webviz_subsurface/plugins/_structural_uncertainty/views/dialog.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+import dash_bootstrap_components as dbc
+import webviz_core_components as wcc
+from dash import html
+
+
+def dialog_layout(
+    uuid: str,
+    dialog_id: str,
+    title: str,
+    children: List,
+    size: str = "sm",
+) -> wcc.Dialog:
+
+    return wcc.Dialog(
+        # style={"marginTop": "20vh"},
+        children=children,
+        id={"id": uuid, "dialog_id": dialog_id, "element": "wrapper"},
+        max_width=size,
+        title=title,
+        draggable=True,
+    )
+
+
+def open_dialog_layout(uuid: str, dialog_id: str, title: str) -> dbc.Button:
+    return html.Div(
+        children=html.Button(
+            title,
+            id={"id": uuid, "dialog_id": dialog_id, "element": "button-open"},
+        ),
+    )
+
+
+def clear_all_apply_dialog_buttons(
+    uuid: str, dialog_id: str, apply_disabled: bool = True
+) -> html.Div:
+    return html.Div(
+        children=[
+            dbc.Button(
```

Comment:
These buttons can be changed into regular html.Buttons to remove the bootstrap dependency here as well. I did that in the ParameterFilter and the buttons looked and behaved the same

Reply:
Thanks. Will fix in a separate pr :) #974